README.md

```diff
@@ -1,15 +1,117 @@
 # GraphDB AWS Terraform Module
 
-TBD: Intro
+This Terraform module allows you to provision an GraphDB cluster within a Virtual Private Cloud (VPC). The module provides a flexible way to configure the cluster and the associated VPC components.
```

Comment:
We could mention that this module "Implements the GraphDB reference architecture on AWS" - https://graphdb.ontotext.com/documentation/10.4/aws-deployment.html